### PR TITLE
Examples for API ref

### PR DESCRIPTION
**Pull request by ronny-mysten**
## Description 

Examples for rpc api ref. Resubmitting PR to capture updates made to Sui repo since previous submission. A few additional examples added since then, too. 

## Test Plan 

Locally

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-open-rpc/src/examples.rs [hunk 1]
```diff
@@ -568,10 +635,218 @@ impl RpcExampleProvider {
...
+        let address = SuiAddress::from(ObjectID::new(self.rng.gen()));
+
+        let result = Balance {
+            coin_type: "0x2::sui::SUI".to_string(),
+            coin_object_count: 15,
+            total_balance: 3000000000,
+            locked_balance: HashMap::new(),
+        };
+        
+        //println!("{}", serde_json::to_string_pretty(&y).unwrap());
```
Comment (wlmyng) [inline crates/sui-open-rpc/src/examples.rs:687]: nit: println

File: crates/sui-open-rpc/src/examples.rs [hunk 2]
```diff
@@ -568,10 +635,218 @@ impl RpcExampleProvider {
...
+            )]
+        )
+    }
+
+    fn suix_get_total_supply(&mut self) -> Examples {
+
+        let mut coin = ObjectID::new(self.rng.gen()).to_string();
+        coin.push_str("::acoin::ACOIN");
+
+        let result = Value {
```
Comment (wlmyng) [inline crates/sui-open-rpc/src/examples.rs:836]: question: are we not able to use `Supply` here?

**Review by wlmyng — APPROVED** [review]
thank you! this is money moves

File: crates/sui-open-rpc/src/examples.rs (hunk 1 above)
Comment (ronny-mysten) [reply crates/sui-open-rpc/src/examples.rs]: oops

**Pull request merged**